tasks/main.yml

```
@@ -8,7 +8,7 @@
   become: false
 
 - name: Ensure all configured dotfiles are links.
-  command: "ls -F {{ dotfiles_home }}/{{ item }}"
+  command: "ls -F {{ dotfiles_home }}/'{{ item }}'"
```

I would say that surrounding the whole path with double-quotes would be better. Then it's also possible to have spaces in `{{ dotfiles_home }}`. Double-quotes enable variables like `$USER` to be resolved when needed. 
```suggestion
  command: >-
    ls -F "{{ dotfiles_home }}/{{ item }}"
```

---

The reason for only quoting the `{{ item }}` is that default value for `{{ dotfiles_home }}` is `~` and if the `~` is within the quotes then it doesn't resolve to the home directory.

For example the below doesn't work

```bash
ls -F "~/<file>"
ls: ~/<file>: No such file or directory
```

Where this does work

```bash
ls -F ~/"<file>"
/Users/user/<file>
```